### DISCUSSION TITLE
Can we import local directories?

### DISCUSSION THREAD

**Discussion by vdegenne** [LazyVim/LazyVim#4956] [Q&A] [answered]
Is it possible to import local directories for when we code a plugin? I mean something like this:
```lua
return {
  "$HOME/gits/my-lua-plugin.nvim/"
}
```

**Comment by dpetka2001** [comment, marked as answer]
See [lazy.nvim docs](https://lazy.folke.io/spec#spec-source). I only use `dev = true` and `config.dev.path`, but you can also configure a separate path other then `config.dev.path` with using `dir = "/some/local/dir/"` depending on what fits best your needs.

**Reply by vdegenne** [reply]
Ah! that's what `dir` is for. Thanks 🙏

**Comment by JustAnAverageGuy** [comment]
Hi. When specifying local plugins, can we specify the branch as well. Because when I am trying to use the branch option to choose the branch, it is not working.

**Reply by dpetka2001** [reply]
When you switch to the branch of your choosing in your repo it gets picked up by `lazy.nvim` automatically.
